src/service/poc_mobile.proto

```diff
@@ -7,6 +7,70 @@ import "mapper.proto";
 import "hex_boosting.proto";
 import "service_provider.proto";
 
+message hex_usage_counts_req_v1 {
+  // H3 resolution 10 tile
+  uint64 hex = 1;
+  // Average count of unique helium mobile subscribers active in the hex during
+  // the previous 7 days ( inclusive of discovery mapping subscribers )
+  uint64 helium_mobile_subscriber_avg_count = 2;
+  // Average count of unique helium mobile discovery mapping subscribers active
+  // in the hex during the previous 7 days
+  uint64 helium_mobile_disco_mapping_avg_count = 3;
```

Comment:
probably shouldn't make these helium-mobile specific, right? `carrier_subscriber_avg_count` and such?

Reply:
renamed to service_provider_subscriber_avg_count and disco_mapping_avg_count